test/integration/api/datasets.js

```diff
@@ -62,15 +62,15 @@ describe('datasets and entities', () => {
                   }))))));
     });
 
-    describe('projects/:id/datasets/:dataset/download GET', () => {
```

Comment:
add a test for checking unpublished dataset can't be linked to a form attachment